src/py/crankshaft/crankshaft/clustering/kmeans.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.cluster import KMeans
+import plpy
+
+def kmeans(query, no_clusters, no_init=20):
```

Comment:
Keep it like that, we could expose that later at camshaft level.

And thanks a lot for the explanation 😄 .